.travis.yml

```diff
@@ -29,8 +29,9 @@ jobs:
     - stage: Documentation
       if: (branch = develop) AND (type = push)
       script:
+        # Exit devon4j folder
+        - cd ..
         # Clone repositories
-        - git clone -b develop --single-branch https://${GH_REPO_SOURCE}
```

Comment:
@jambulud Excellent improvement 👍 You save resources and speed up CI this way.
Could you also remove the `GH_REPO_SOURCE` variable if we already cleanup the travis file.
Are you doing this for all repos in `devonfw`? I never wrote this stuff and assume it has been copy&pasted to all major repos.

Reply:
@hohwille Removed unused variable

I just did this for devon4ng (and now devon4j) because I saw that devonfw-guide build process was not being triggered by the updates in the documentation